packages/shared/src/DropMarker/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow strict
```

Comment:
please don't use the index.js name

I cant find any files using search for filenames if we have 100 files named index.js